qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -739,7 +739,14 @@ def on_load_finished(self, tab, ok):
             start = config.val.colors.tabs.indicator.start
             stop = config.val.colors.tabs.indicator.stop
             system = config.val.colors.tabs.indicator.system
-            color = utils.interpolate_color(start, stop, 100, system)
+            try:
+                url_str=self.widget.tab_url(idx).toString()
+                if url_str.find("qute://back") != -1 :
+                    color = config.val.colors.tabs.indicator.suspended
+                else :
+                    color = utils.interpolate_color(start, stop, 100, system)
+            except :
```

Comment:
Ah, broad excepts like this are really frowned on without a good reason. What errors are you expecting here?
Also you have extra spaces before your colons.